docs/installation/manual/introduction.md

```diff
@@ -0,0 +1,88 @@
+
+`/path/to/file_being_modified`
+
+**Before**:
+> 174 | Here is what line 174 in the file looked like before
+
+> 175 | And here is what the following line looked like in the file before
+
+**After**:
+> 174 | Here is what line 174 should look like after modification
+
+> 175 | And here is what the following line should look like after modification: VARIABLE
+
+- `VARIABLE`: `some_value`, perhaps with an explanation of why
+
+It should be noted that configuration files and the like are subject to change by the organizations that maintain their respective applications; this guide generally recommends installing the latest version of these applications, as these generally include security updates. It is expected that the implementer will be able to search through a file and find specific lines in the case where the maintaining organization has moved it in a subsequent patch. In most cases, configuration files will be provided outright to avoid these scenarios.
+
+### Adding or Replacing Files
+
+When a file needs to be added or replaced, it will be described in three sections:
+
+- A line that describes the path to the file, as well as the owner, group, and umask for the file; it is assumed that the person following the guide will use `chmod` and `chown` appropriately to apply the owner, group, and umask
+- The entire contents of the file in a code block, including any portions that need to be replaced with specific values
+- Those replacement values
+
+`/the/path/to/some/file.php | owner:group/umask`
+```php
+<?php
+
+/**
+ * Here is the entire contents of a PHP file
+ */
+function do_something($to_this) {
+  $to_this += THE_NUMBER_TO_ADD_TO_THIS;
+}
+
+?>
+```
+- `THE_NUMBER_TO_ADD_TO_THIS`: 12, perhaps with an explanation of why, or other numbers that may be appropriate
+
+### Troubleshooting
+
+The most common issues you will likely run into when manually provisioning a server are:
+
+- Files or directories are not owned by the user who needs access to them, and can therefore not be written to. Check the ownership of files using `ls -la`, and ensure their ownership using `chown USER` for files, and `chown -R USER` for directories
+- Replacement variables were left in place in files specified by the guide. Ensure any replacement variables such as server addresses and passwords are swapped out when writing files to the server
+
+For any other issues, don't hesitate to email the [mailing list](islandora@googlegroups.com) to ask for help. If you think that a part of the installation documentation is incorrect or could be improved, please create an issue in the [http://github.com/Islandora/documentation/issues](documentation issues queue) and give it a `documentation` tag. Bear in mind that this guide is built for Ubuntu 18.04 and attempts to give generalized instructions; you will likely naturally encounter situations where your own environment needs to differ from the guide.
```

Comment:
The "documentation issues queue" link is reversed. It should be `[documentation issues queue](http://github.com/Islandora/documentation/issues)`